src/OpenTK/Platform/DisplayDeviceBase.cs

```diff
@@ -32,24 +32,22 @@ internal abstract class DisplayDeviceBase : IDisplayDeviceDriver
         protected readonly List<DisplayDevice> AvailableDevices = new List<DisplayDevice>();
         protected DisplayDevice Primary;
 
-        public abstract bool TryChangeResolution(DisplayDevice device, DisplayResolution resolution);
-        public abstract bool TryRestoreResolution(DisplayDevice device);
+        public abstract bool TryChangeResolution (DisplayDevice device, DisplayResolution resolution);
+        public abstract bool TryRestoreResolution (DisplayDevice device);
+        public abstract Vector2 GetDisplayScaling (DisplayIndex displayIndex);
 
         // Gets the DisplayDevice that corresponds to the specified index.
-        public virtual DisplayDevice GetDisplay(DisplayIndex index)
+        public virtual DisplayDevice GetDisplay (DisplayIndex index)
```

Comment:
As a small style nitpick we tend to not have spaces between the method name and the parens.